.env.development.local.example

```diff
@@ -1,2 +1,2 @@
-#ACCOUNT_ID=nf-payments.near
+#ACCOUNT_IDS=nf-payments.near,example.near
```

Comment:
is this needed?

Reply:
@nearken For the commented out `ACCOUNT_IDS` in the `.env`, I find comments like that to be helpful hints to remind (or instruct) someone what kinds of values can go there (so that they don't need to look it up elsewhere).

I agree about reorganizing folders.

Reply:
Hints may makes more sense in the README. I was initially confused by having this in the .env because it seems appears that it is used as an environment variable (but it wasn't)

Reply:
My view is that it's a balance. Sometimes READMEs get so long that people then don't even follow them closely enough. We don't want that.

We also don't want so many comments in `.env` files such that those are clunky. 

But in this case the `.env` is so small, this seems super convenient.

What do you mean "appears that it is used as an environment variable (but it wasn't)"?

(The `ACCOUNT_IDS` line does get used if the `#` gets deleted from the beginning of the line, making it no longer a comment.)